### ISSUE TITLE
Connecting to Codespace fails with authentication issue

### ISSUE THREAD

**Issue by jlynch630** [MicrosoftDocs/vsonline#561]
After having issues with an existing codespace, I deleted it and created a new one. However, when opening the new one, I was presented with an error that says I don't own the codespace. Trying with a new codespace, I get the same result. 

In Visual Studio 2019 it looks like this:
![image](https://user-images.githubusercontent.com/1965320/83594601-7e975f80-a514-11ea-8a4f-06abbcda2ffb.png)
Clicking "Sign-In" opens a dialog with "Acquiring user credentials..." then re-displays the banner.

In Visual Studio Online, it works about half the time, the other half it throws an error saying that it timed out waiting for the extension host.

**Comment by jkeech** [comment]
@jlynch630, are you using the Windows Codespaces private preview and connecting from Visual Studio? It looks like the sign-in credentials might be stale on the client side. Can you try two things?

1) Click on your profile in the top right corner and open Account Settings. Then make sure none of the accounts listed say they need to refresh credentials. If any of them say they need to refresh credentials, go ahead and do that.
![image](https://user-images.githubusercontent.com/746020/84050193-3fc53780-a962-11ea-86ef-5874cbf39c78.png)

2) Can you double check the value of the Live Share -> Authentication -> User account option in Tools -> Options? It should look something like this where it is configured to use the personalization account.
![image](https://user-images.githubusercontent.com/746020/84049964-f1179d80-a961-11ea-8015-b96611cce2b2.png)

**Comment by jlynch630** [comment]
Yep, I'm on the private preview. I thought it might be a server-side issue but VS Code is working more reliably now. I've tried signing in and out and refreshing my credentials to no avail, and the same issue happens on a fresh install on a different machine. 

Live share was set to personalization account, but, oddly enough, setting it to external account allowed me to connect to the codespace just once, even though I was never prompted to sign in on the new machine. In subsequent attempts the same error appears whether I'm on personalization or external.

**Comment by nandiniYeltiwar** [comment]
@jlynch630 Can you try suspending and connecting to the environment? You can do it from https://online.visualstudio.com/environments or from VS Code

**Comment by jlynch630** [comment]
Unfortunately the issue persists across all codespaces, even after suspending them (resuming the codespace actually totally works fine in VS itself). Even in a new plan I made in west us, the issue still occurs

**Comment by nandiniYeltiwar** [comment]
@jlynch630 We are investigating the issue which currently does impact all codespaces. Just to confirm, did resuming the workspace from VS worked and were you unblocked from using that codespace?

**Comment by jlynch630** [comment]
Sorry that last comment of mine was poorly worded @nandiniYeltiwar :). Even after resuming the workspace I still get the same sign-in error.

**Comment by nandiniYeltiwar** [comment]
Thanks for confirming @jlynch630. I will follow up once we have fixed the issue.

**Comment by lostintangent** [comment]
@nandiniYeltiwar Any updates here?
